app/src/main/java/edu/uc/hornunjb/weatherwear/dto/WeatherResponse.kt

```diff
@@ -0,0 +1,57 @@
+package edu.uc.hornunjb.weatherwear.dto
+
+import com.google.gson.annotations.SerializedName
+import java.lang.StringBuilder
+import java.util.ArrayList
+
+/**
+ * @author meyer3cj
+ * This class will gather the information from the other classes
+ */
```

Comment:
I wrote some documentation for this class. The class will take the data from the other classes.